kube-core/Cargo.toml

```diff
@@ -26,27 +26,19 @@ schema = ["schemars"]
 kubelet-debug = ["ws"]
 
 [dependencies]
-serde = { version = "1.0.130", features = ["derive"] }
-serde_json = "1.0.68"
-thiserror = "1.0.29"
-form_urlencoded = "1.0.1"
-http = "0.2.5"
-json-patch = { version = "1.0.0", optional = true }
-once_cell = "1.8.0"
-chrono = { version = "0.4.19", default-features = false, features = ["clock"] }
-schemars = { version = "0.8.6", optional = true }
-
-[dependencies.k8s-openapi]
-version = "0.21.0"
-default-features = false
-features = []
-
-[dev-dependencies.k8s-openapi]
-version = "0.21.0"
-default-features = false
-features = ["latest"]
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true }
+thiserror = { workspace = true }
+form_urlencoded = { workspace = true }
+http = { workspace = true }
+json-patch = { workspace = true, optional = true }
+once_cell = { workspace = true }
+chrono = { workspace = true, features = ["clock"] }
+schemars = { workspace = true, optional = true }
+k8s-openapi= { workspace = true, features = [] }
```

Comment:
```suggestion
k8s-openapi= { workspace = true }
```

Reply:
done in https://github.com/kube-rs/kube/pull/1435/commits/cfe86580667dd417258d25917f1d290e956c3d06